db/migrate/20150324112658_add_onboarding_fields_to_order_permissions.rb

```diff
@@ -0,0 +1,13 @@
+class AddOnboardingFieldsToOrderPermissions < ActiveRecord::Migration
```

Comment:
We might want to prefer using :after clauses to ensure that our schema.rb stays the same.